Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/security/helper/JwtHelper.java

```diff
@@ -38,32 +38,47 @@
 
 import javax.servlet.http.Cookie;
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import java.nio.charset.StandardCharsets;
 import java.security.*;
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.time.Duration;
+import java.time.Instant;
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.util.Date;
+import static org.apromore.portal.util.AssertUtils.notNullAssert;
 
 public class JwtHelper {
 
     public static final String JWT_KEY_SUBJECT_USERNAME = "sub";
+    public static final String JWT_KEY_ISSUED_AT = "iat";
+    public static final String JWT_EXPIRY_TIME = "exp";
```

Comment:
These constants duplicate JWTClaimsSet.SUBJECT_CLAIM, ISSUED_AT_CLAIM and EXPIRATION_TIME_CLAIM.  See line 84+ of the source code at https://github.com/felx/nimbus-jose-jwt/blob/master/src/main/java/com/nimbusds/jwt/JWTClaimsSet.java